aikau/src/main/resources/alfresco/prototyping/Preview.js

```diff
@@ -285,6 +285,7 @@ define(["dojo/_base/declare",
             {
                this.rootPreviewWidget.destroyRecursive(false);
             }
+            this.previewNode.innerHTML = this.message("loading.preview.message");
```

Comment:
We should be cautious about ever using `innerHTML` over `textContent`. There's no real harm here given the source, but it's worth mentioning.